src/cli/commands/task/list.ts

```diff
@@ -16,13 +16,17 @@ export default defineCommand({
   },
   async run({ args }) {
     const cwd = resolve((args.dir || args.cwd || ".") as string);
-    const tasks = await listTasks({ cwd, buildDir: ".nitro" });
+    const nitro = await createNitro({ rootDir: cwd });
```

Comment:
I think you can use lighter `loadConfig` to load nitro config only

Reply:
I'll take a look at it today

Reply:
i used the loadOptions() because i couldn't find an example on the loadConfig function.
I hope that's okay.